tweetflask.py

```diff
@@ -57,19 +69,27 @@ def get_tweet(jsondata, user, mood):
 
 
 def errors(status, detail):
-    aboutur = "https://developer.oregonstate.edu/documentation/error-reference"
+    error_base_url = "https://developer.oregonstate.edu/documentation"
+    reference_error = "/error-reference"
+    code = f'1{status}'
     if status is '400':
-        title = "Username is invalid"
+        title = "Bad Reques"
+    if status is '401':
+        title = "Authentication faild"
+    elif status is '404':
+        title = "Page not found"
+    elif status is '500':
+        title = "Unexpected Internal error"
```

Comment:
Instead of using a multi-level if-else block, could you use a [dictionary](https://docs.python.org/3.7/tutorial/datastructures.html#dictionaries) to optimize the logic?